ansible_collections/arista/cvp/molecule/cv_facts_v3/test_cv_facts_v3.yml

```diff
@@ -0,0 +1,244 @@
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices != []
+
+    - name:  Collect devices facts from {{inventory_hostname}} with verbose short
+      arista.cvp.cv_facts_v3:
+        facts:
+          - devices
+        regexp_filter: "{{DEVICE_NAME}}"
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts for device spine1 with verbose short
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.failed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices[0].hostname == "spine1"
+
+    - name: Collect devices facts from {{inventory_hostname}} with verbose long
+      arista.cvp.cv_facts_v3:
+        facts:
+          - devices
+        regexp_filter: "{{DEVICE_NAME}}"
+        verbose: long
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts for device spine1 with verbose long
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.failed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_images == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices[0].hostname == "spine1"
+          - CV_FACTS_V3_RESULT.data.cvp_devices[0].deviceStatus == "Registered" # we get this key only with verbose long
+
+    - name: Collect images facts from {{inventory_hostname}}
+      arista.cvp.cv_facts_v3:
+        facts:
+          - images  # collects all images facts
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_images != []
+
+    - name: Collect images facts from {{inventory_hostname}} filtered by name
+      arista.cvp.cv_facts_v3:
+        facts:
+          - images
+        regexp_filter: "{{IMAGE_NAME}}" # collects specific image facts
+      register: CV_FACTS_V3_RESULT
+
+    - name: Check cv_facts_v3 result
+      assert:
+        that:
+          - CV_FACTS_V3_RESULT.changed == false
+          - CV_FACTS_V3_RESULT.data.cvp_configlets == []
+          - CV_FACTS_V3_RESULT.data.cvp_containers == []
+          - CV_FACTS_V3_RESULT.data.cvp_devices == []
+          - CV_FACTS_V3_RESULT.data.cvp_tasks == []
+          - CV_FACTS_V3_RESULT.data.cvp_images != []
```

Comment:
Can we not get the image name here and assert that?

Reply:
we can do something like, `CVP_FACTS_V3.data.cvp_images['EOS-4.25.4M.swi']` and make sure that's not returning an error

Reply:
Fixed!